src/vmm/src/builder.rs

```diff
@@ -1258,6 +1263,23 @@ pub mod tests {
             .is_some());
     }
 
+    pub(crate) fn insert_entropy_device(
+        vmm: &mut Vmm,
+        cmdline: &mut Cmdline,
+        event_manager: &mut EventManager,
+        entropy_config: EntropyDeviceConfig,
+    ) {
+        let mut builder = EntropyDeviceBuilder::new();
+        let entropy = builder.build(entropy_config).unwrap();
+
+        assert!(attach_entropy_device(vmm, cmdline, event_manager, &entropy).is_ok());
```

Comment:
Nit it is usually better to match the specific value e.g. assert_eq!(..., Ok(...)).

Reply:
Here it would be doable, if `StartMicrovmError` implemented `PartialEq`